src/content/structured/components/breadcrumbs/code.mdx

```diff
@@ -51,3 +51,206 @@ export const snippetsDefault = [
 </ComponentPreview>
 
 <ComponentDetails component="ic-breadcrumb-group" />
+
+## Variants
+
```

Comment:
Sorry I forgot to add it to the list, but could we add a code example for react router?